### ISSUE TITLE
All changefeeds get stuck, when upstream has tpcc and other DDL workload, let cdc server restart 

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/ticdc#2111]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- 3x cdc server, 3x TiKV, 3x changefeed.
- Workload in upstream is lightning loading DDL( table count: 8002), and tpcc prepare/run(table count: 8).
-  Restart pd, then restart CDC "tiup cluster restart <clustername> -R cdc" for 4 times one by one in 1 minutes.
![image](https://user-images.githubusercontent.com/78345569/122708460-d1ae9280-d28e-11eb-8277-671ab5882d5a.png)
![image](https://user-images.githubusercontent.com/78345569/122708540-f60a6f00-d28e-11eb-9e9b-6a3ad3e7c57b.png)
![image](https://user-images.githubusercontent.com/78345569/122708617-2520e080-d28f-11eb-84a6-f1640da8d125.png)
![image](https://user-images.githubusercontent.com/78345569/122708649-34a02980-d28f-11eb-9ba2-7a8eb86b2908.png)

2. What did you expect to see?

- All changefeed works normally, no error log.

3. What did you see instead?

- All changefeed get stuck, one capture is down, cdc log has many error.
- Restart all cdc server again , all cf still donot work, one of capture get down again.
Error log in cdc.log:
![image](https://user-images.githubusercontent.com/78345569/122709135-3b7b6c00-d290-11eb-9ed3-202f2735fcda.png)
![image](https://user-images.githubusercontent.com/78345569/122709159-4a621e80-d290-11eb-82b7-eafa5de5a13c.png)

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
       Release Version: v5.1.0
Edition: Community
Git Commit Hash: 4ca5d03fbe2fe1d2106a29a38b4c99be66b36eb1
Git Branch: heads/refs/tags/v5.1.0
UTC Build Time: 2021-06-17 07:24:39
GoVersion: go1.16.4
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false |

        ```

    - TiCDC version (execute `cdc version`):

        ```
       [release-version=v5.1.0] [git-hash=a91a10bb5cd52ee22e56217402003a596a1b5f2b] [git-branch=heads/refs/tags/v5.1.0] [utc-build-time="2021-06-17 07:20:55"]
        ```

**Comment by Tammyxia** [comment]
Fixed and merged in 5.1, pr https://github.com/tikv/tikv/issues/10411

**Comment by overvenus** [comment]
> Fixed and merged in 5.1, pr [tikv/tikv#10411](https://github.com/tikv/tikv/issues/10411)

Can we close this issue?

**Comment by Tammyxia** [comment]
Upgrade KV & cdc to v5.1.0 06-23, this issue is not reproduced.
